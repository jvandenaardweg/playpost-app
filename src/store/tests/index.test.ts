import { persistor, store } from '../index';

/**
 * These tests are just basic tests to make sure the methods keep returning the thing we expect to
 */
describe('store/index', () => {
  it('store.getState() should return the initial store state', () => {
    expect(store.getState()).toMatchSnapshot();
  });

  it('persistor.getState() should return the initial store state', () => {
    expect(persistor.getState()).toMatchSnapshot();
  });

});
