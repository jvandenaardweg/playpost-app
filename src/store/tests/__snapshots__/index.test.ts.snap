// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`store/index persistor.getState() should return the initial store state 1`] = `
Object {
  "bootstrapped": false,
  "registry": Array [
    "root",
  ],
}
`;

exports[`store/index store.getState() should return the initial store state 1`] = `
Object {
  "_persist": Object {
    "rehydrated": false,
    "version": 7,
  },
  "audiofiles": Object {
    "downloaded": Array [],
    "error": "",
    "isLoading": false,
  },
  "auth": Object {
    "error": "",
    "errorRequestResetPasswordToken": "",
    "errorUpdatePassword": "",
    "isLoading": false,
    "isLoadingResetPassword": false,
    "isLoadingUpdatePassword": false,
    "token": "",
  },
  "player": Object {
    "audiofile": null,
    "currentArticleId": "",
    "error": "",
    "errorCreateAudiofile": "",
    "isCreatingAudiofile": false,
    "isDownloadingAudiofile": false,
    "isLoadingCreateAudiofile": false,
    "playbackState": "none",
    "previousArticleId": "",
    "track": Object {
      "artist": "",
      "contentType": "",
      "description": "",
      "duration": 0,
      "id": "",
      "title": "",
      "url": "",
    },
  },
  "playlist": Object {
    "error": "",
    "isLoading": false,
    "isLoadingArchiveItem": false,
    "isLoadingCreateItem": false,
    "isLoadingFavoriteItem": false,
    "isLoadingReOrderItem": false,
    "isLoadingUnArchiveItem": false,
    "isLoadingUnFavoriteItem": false,
    "items": Array [],
  },
  "subscriptions": Object {
    "error": "",
    "errorValidateSubscriptionReceipt": "",
    "isLoadingRestore": false,
    "isLoadingSubscriptions": false,
    "isLoadingUpgrade": false,
    "isLoadingValidateSubscriptionReceipt": false,
    "subscriptions": null,
    "validationResult": null,
  },
  "user": Object {
    "details": null,
    "deviceLocale": "en",
    "error": "",
    "errorSaveSelectedVoice": "",
    "isLoading": false,
    "isLoadingDelete": false,
    "isLoadingSaveSelectedVoice": false,
    "isLoadingUpdateEmail": false,
    "isLoadingUpdatePassword": false,
    "playbackSpeed": 1,
  },
  "voices": Object {
    "downloadedVoicePreviews": Array [],
    "error": "",
    "isLoading": false,
    "isLoadingLanguages": false,
    "languages": Array [],
  },
}
`;
